src/Files.App/Views/Properties/MainPropertiesPage.xaml.cs

```diff
@@ -98,6 +98,20 @@ private async void Page_Loaded(object sender, RoutedEventArgs e)
 			}
 		}
 
+		private void MainPropertiesPage_SizeChanged(object sender, SizeChangedEventArgs e)
+			=> UpdateDialogLayout();
```

Comment:
Coding convention in Files is to use brackets

Reply:
Does that mean that we should not lambda expression for methods in anywhere?

Reply:
There might be exceptions but not in this context.